VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -263,6 +263,7 @@
 from PhysicsTools.Heppy.analyzers.objects.JetAnalyzer import JetAnalyzer
 JetAna = JetAnalyzer.defaultConfig
 JetAna.calculateSeparateCorrections = True # CV: needed for ttH prompt lepton MVA
+JetAna.lepSelCut = lambda lep : (abs(lep.pdgId()) == 11 and lep.relIso03 < 0.4) or (abs(lep.pdgId()) == 13 and lep.relIso04 < 0.4)
```

Comment:
what is this meant for? we already have a selection for the selectedLeptons used here.
The only difference seem to be you do not OR with miniIsoltion, is that the purpose? remove the miniIsolation or?